wdl2cwl/main.py

```diff
@@ -223,7 +223,7 @@ def load_wdl_workflow(self, obj: WDL.Tree.Workflow) -> cwl.Workflow:
             cwl.WorkflowOutputParameter(
                 id=f"{wf_name}.{output.id}",
                 type=output.type,
-                outputSource=output.outputBinding.glob.replace(".", "/")[2:-1]
+                outputSource=output.outputBinding.glob
```

Comment:
See https://github.com/common-workflow-lab/wdl-cwl-translator/pull/155/files#diff-b5a65c2befbbd0b6453805272729b622f41b2be683f8258432e08f4370332265R226 for a different opinion :-)